packages/unlock-js/src/PublicLock/v12/withdrawFromLock.js

```diff
@@ -12,12 +13,23 @@ import utils from '../../utils'
  * @param {function} callback invoked with the transaction hash
  */
 export default async function (
-  { lockAddress, beneficiary, amount = '0', decimals = 18 },
+  { lockAddress, beneficiary, amount = '0', decimals, erc20Address },
   transactionOptions = {},
   callback
 ) {
   const lockContract = await this.getLockContract(lockAddress)
-  const tokenAddress = await lockContract.tokenAddress()
+  const tokenAddress = erc20Address || (await lockContract.tokenAddress())
```

Comment:
You are not using `tokenAddress` below?

Reply:
the idea here is to retrieve `tokenAddress` when is not passed as prop

Reply:
oh i see what you mean